tests/acceptance/features/webUIPreview/imageMediaViewer.feature

```diff
@@ -118,3 +118,39 @@ Scenario Outline: preview of image files with media viewer is possible
       | path | simple-empty-folder |
     When the public uses the webUI to access the last public link created by user "user1"
     Then the preview image of file "testavatar.jpg" should not be displayed in the file list view on the webUI
+
+  @issue-ocis-1490
+  Scenario: preview of image in file list view for .jpeg format file
+    Given user "user1" has uploaded file "testavatar.jpg" to "testavatar.jpeg"
+    And user "user1" has logged in using the webUI
+    When the user browses to the files page
+    Then the preview image of file "testavatar.jpeg" should be displayed in the file list view on the webUI
+    When the user views the file "testavatar.jpeg" in the media viewer using the webUI
+    Then the file "testavatar.jpeg" should be displayed in the media viewer webUI
+
+  @issue-ocis-1490
+  Scenario: preview of image in file list view for .ogg format file
+    Given user "user1" has uploaded file "testavatar.jpg" to "testavatar.ogg"
```

Comment:
I'm not sure if we really expect this thing to work, because now we have an jpeg image that has the extenion `ogg` but it is not an `ogg` file.
@kulmann would we expect the media viewer still to generate a preview?
Maybe better to upload a real `ogg` file.

similar in other tests

Reply:
Right now the media viewer operates based on the file extension, so for an `.ogg` file it will try to render it in an html `<video>` tag.

Reply:
Sample files with required formats are uploaded